content/learning-paths/servers-and-cloud-computing/rme-cca-basics/rme-cca-fvp.md

```diff
@@ -0,0 +1,186 @@
+
+In this learning path, you will learn how to build and run the reference integration software stack for Arm CCA which demonstrates support for Arm's RME architecture feature. You will also learn how to create a realm that runs a guest linux kernel. 
+
+## Build the docker container
+
+You can build the Arm CCA reference software stack in a docker container which contains all the build dependencies. 
+Install [docker engine](/install-guides/docker/docker-engine) on your machine.
+
+Clone the repository that contains the docker container file and utility scripts:
+
+```console
+git clone --branch AEMFVP-A-RME-2023.09.29 https://git.gitlab.arm.com/arm-reference-solutions/docker.git
+```
+Build the docker container:
+
+```console
+cd docker
+./container.sh build
+```
+The script `container.sh` defines the docker file and image name used to create the container.
+
+Confirm that the docker container image was built successfully:
+
+```
+docker image list
+```
+
+The expected output is:
+
+```output
+REPOSITORY        TAG       IMAGE ID       CREATED       SIZE
+aemfvp-builder    latest    2fa7ce18f57a   7 mins ago    1.83GB
+```
+
+Create a directory on your host machine to store the software source files. Then run the container and mount this directory inside the container:
+
+```console
+mkdir ~/cca-stack
+./container.sh -v ~/cca-stack run
+```
+
+You should see the following output:
+
+```output
+Running docker image: aemfvp-builder ...
+ubuntu@ip-172-16-0-235:/$
+```
+
+You are now inside the root directory of the `aemfvp-builder` container and ready to build the software stack.
+
+## Build the reference Arm CCA software stack
+
+You can build the Arm CCA software stack in your running container using a manifest file. Inspect the [manifest file](https://git.gitlab.arm.com/arm-reference-solutions/arm-reference-solutions-manifest/-/blob/AEMFVP-A-RME-2023.09.29/pinned-aemfvp-a-rme.xml) to view all the component repositories needed to build this reference stack. 
+
+Inside the running container, change directory into the mounted directory.
+Use the repo tool and the manifest file to download the software stack:
+
+```console
+cd ~/cca-stack
+repo init -u https://git.gitlab.arm.com/arm-reference-solutions/arm-reference-solutions-manifest.git -m pinned-aemfvp-a-rme.xml -b refs/tags/AEMFVP-A-RME-2023.09.29
+repo sync -c -j $(nproc) --fetch-submodules --force-sync --no-clone-bundle
+```
+
+Patch the linux kernel Kconfig file for arm64 targets. This patch enables extending the bootloader provided command line arguments. It is required to speed up execution of the software stack. 
+
+```console
+cd linux
+wget https://raw.githubusercontent.com/ArmDeveloperEcosystem/arm-learning-paths/main/content/learning-paths/servers-and-cloud-computing/rme-cca-basics/kconfig.patch
```

Comment:
[This link](https://raw.githubusercontent.com/ArmDeveloperEcosystem/arm-learning-paths/main/content/learning-paths/servers-and-cloud-computing/rme-cca-basics/kconfig.patch) did not work for me.

Reply:
Unfortunately this link will only work after the content is merged. This is because the file to be uploaded is part of the pull request.

Reply:
@KhaledBenkrid - for the review, please use this staking link instead: https://raw.githubusercontent.com/pareenaverma/arm-learning-paths/rme-cca-branch/content/learning-paths/servers-and-cloud-computing/rme-cca-basics/kconfig.patch